pkg/controllers/cassandra/nodepool/nodepool.go

```diff
@@ -0,0 +1,131 @@
+package nodepool
+
+import (
+	"fmt"
+
+	v1alpha1 "github.com/jetstack/navigator/pkg/apis/navigator/v1alpha1"
+	"github.com/jetstack/navigator/pkg/controllers/cassandra/util"
+	apps "k8s.io/api/apps/v1beta1"
+	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	appslisters "k8s.io/client-go/listers/apps/v1beta1"
+	"k8s.io/client-go/tools/record"
+)
+
+type Interface interface {
+	Sync(*v1alpha1.CassandraCluster) error
+}
+
+type defaultCassandraClusterNodepoolControl struct {
+	kubeClient        kubernetes.Interface
+	statefulsetLister appslisters.StatefulSetLister
```

Comment:
(nit) in the `cassandra_control.go` file, you call this `statefulSet` instead of `statefulset`

Reply:
Done.